conf/modules/actuators_dummy.xml

```diff
@@ -4,9 +4,12 @@
   <doc>
     <description>
       Dummy actuators driver
+      This module prevents autoloading of actuators by fixedwing firmware
     </description>
   </doc>
   <header/>
-  <makefile/>
+  <makefile>
+    <configure name="ACTUATORS"/>
```

Comment:
Is there a nice way to test that `ACTUATORS` is either null, undefined or equal to `none` on this line: https://github.com/paparazzi/paparazzi/blob/master/conf/firmwares/fixedwing.makefile#L33 ?